### PR TITLE
Add formatted strings not supported note for BCL and DBGL

### PR DESCRIPTION
**Pull request by helpshift-tanuj**

### CODE REVIEW

File: docs/sdkx-unity/tracking-android.mdx [hunk 1]
```diff
@@ -196,6 +196,17 @@ On Custom Issue Fields keys & compatibility
 
 </Admonition>
 
+<Admonition type="info" title="Note">
+
+In the **Breadcrumbs** and **Debug logs** APIs, passing of formatted strings is not supported. It is recommended to pass data only in a simple String format.
+
+For example - 
+
+* "{\"key\":\"value\"}" - ** JSON String, This is not supported **
```
Comment (iTwenty) [inline docs/sdkx-unity/tracking-android.mdx:205]: Can this be a code snippet rather than normal text?

File: docs/sdkx-unity/tracking-android.mdx [hunk 2]
```diff
@@ -196,6 +196,17 @@ On Custom Issue Fields keys & compatibility
...
+* "key - value" - ** Simple string, This is supported **
```
Comment (iTwenty) [inline docs/sdkx-unity/tracking-android.mdx:206]: Same for this

**Review by iTwenty — CHANGES_REQUESTED** [review]

File: docs/sdkx-unity/tracking-android.mdx [hunk 3]
```diff
@@ -196,6 +196,17 @@ On Custom Issue Fields keys & compatibility
 
 </Admonition>
 
+<Admonition type="info" title="Note">
+
+In the **Breadcrumbs** and **Debug logs** APIs, passing of formatted strings is not supported. It is recommended to pass data only in a simple String format.
+
+For example - 
+
```
Comment (helpshift-ps) [inline docs/sdkx-unity/tracking-android.mdx:204]: Remove this line
```suggestion
```

File: docs/sdkx-unity/tracking-ios.mdx [hunk 4]
```diff
@@ -188,6 +188,17 @@ On Custom Issue Fields keys & compatibility
 
 </Admonition>
 
+<Admonition type="info" title="Note">
+
+In the **Breadcrumbs** and **Debug logs** APIs, passing of formatted strings is not supported. It is recommended to pass data only in a simple String format.
+
+For example - 
+
```
Comment (helpshift-ps) [inline docs/sdkx-unity/tracking-ios.mdx:196]: Same here
```suggestion
```

File: docs/sdkx-unreal/tracking.mdx [hunk 5]
```diff
@@ -156,6 +156,17 @@ On Custom Issue Fields keys & compatibility
 
 </Admonition>
 
+<Admonition type="info" title="Note">
+
+In the **Breadcrumbs** and **Debug logs** APIs, passing of formatted strings is not supported. It is recommended to pass data only in a simple String format.
+
+For example - 
+
```
Comment (helpshift-ps) [inline docs/sdkx-unreal/tracking.mdx:164]:
```suggestion
```

File: docs/sdkx_android/tracking.mdx [hunk 6]
```diff
@@ -148,6 +148,17 @@ Compatibility table for type and values:
 | date       | number  | Epoch time. For example - **Date.now()**                        |
 | checkbox   | boolean |                                                                 |
 
+<Admonition type="info" title="Note">
+
+In the **Breadcrumbs** and **Debug logs** APIs, passing of formatted strings is not supported. It is recommended to pass data only in a simple String format.
+
+For example - 
+
```
Comment (helpshift-ps) [inline docs/sdkx_android/tracking.mdx:156]:
```suggestion
```

File: docs/sdkx_ios/tracking.mdx [hunk 7]
```diff
@@ -213,6 +213,17 @@ Helpshift.showConversation(with: self, config: config)
 
 </Tabs>
 
+<Admonition type="info" title="Note">
+
+In the **Breadcrumbs** and **Debug logs** APIs, passing of formatted strings is not supported. It is recommended to pass data only in a simple String format.
+
+For example - 
+
```
Comment (helpshift-ps) [inline docs/sdkx_ios/tracking.mdx:221]:
```suggestion
```

File: docs/sdkx-unity/tracking-android.mdx (hunk 2 above)
Comment (helpshift-tanuj) [reply docs/sdkx-unity/tracking-android.mdx:206]: Its just string formats, Do we need code snippets here?

**Review by helpshift-ps — APPROVED** [review]

**Pull request merged**